spec/controllers/api/v1/rates_controller_spec.rb

```diff
@@ -0,0 +1,27 @@
+require 'rails_helper'
+
+describe Api::V1::RatesController do
+  describe '[GET] #create' do
+    let(:recipe) { create(:recipe) }
+    let(:rate_params) { { vote: 3 } }
+
+    context 'with valid votes' do
+      it 'creates new rate' do
+        expect do
+          post :create, params: { recipe_id: recipe.id, rate: rate_params }
+        end.to change(Recipe, :count).by(1)
+      end
+    end
+
+    context 'with invalid votes' do
+      let!(:recipe) { create(:recipe) }
```

Comment:
Why do you create `recipe` for the second time? Can't you just use recipe created in the first `let`?